web/templ/pages/monster-family.templ

```diff
@@ -0,0 +1,44 @@
+package pages
+
+import "dqix/web/templ/components/base"
+import "dqix/internal/types"
+import "dqix/web/templ/components/links"
+import "dqix/web/templ/utilities"
+
+type MonsterFamilyParams struct {
+	Family          string
+	FamilyTitle     string
+	Monsters        types.MonsterSlice
+	DisplayMode     string
+	LayoutParams    base.LayoutParams
+	SortPathGetter  func(sortField string) templ.SafeURL
+	SortOrderGetter func(sortField string) string
+}
+
+templ MonsterFamilyPage(params MonsterFamilyParams) {
+	@base.Layout(params.LayoutParams) {
+		@MonsterFamilyContent(params)
+	}
+}
+
+templ MonsterFamilyContentWithSideNav(params MonsterFamilyParams) {
+	@base.MainContentWithSidenav(params.Family) {
+		@MonsterFamilyContent(params)
+	}
+}
+
+templ MonsterFamilyContent(params MonsterFamilyParams) {
+	<h1 class="text-3xl font-bold">{ params.FamilyTitle + " Family" }</h1>
+	@MonsterFamilyList(params.Monsters)
+}
+
+templ MonsterFamilyList(monsters types.MonsterSlice) {
+	<ul hx-boost="true" hx-target="#page-content" hx-swap="innerHTML" class="[&>li]:flex [&>li]:items-center">
+		for _, monster := range monsters {
+			<li>
+				// <img src={ monster.ImageSrc() } alt={ monster.GetID() }/>
```

Comment:
I plan to add sourced images to this repo/app later so I just commented this here to make it easier for later.